.travis.yml

```diff
@@ -6,6 +6,21 @@ go:
   - 1.9.x
   - "1.10"
```

Comment:
I know you didn't add this, but since you are tweaking this file could you also change this to `- 1.10.x` instead of `- "1.10"`? Thx!

Reply:
Sure! I'll change this later.